### ISSUE TITLE
Feature Request: Low colour, minimal version of AOD watch face

### ISSUE THREAD

**Issue by jamespaulley** [MilosKozak/AndroidAPS#2189]
There is currently no difference between the Always On Display and Active Display versions of the watch face. Basically it's very hard to tell if the watch face is active or not. Could we have a low/no colour version for the AOD, perhaps with thinner line weights and less detail?

**Comment by dlvoy** [comment]
What is most essential data to display on less detailed version?
- **for sure:** BGV + Trend arrow + measurement age

**Comment by Philoul** [comment]
To complete this request, colors scheme is different between AAPS and Wear watch : 
BG : Green when in target in AAPS, white in Wear watch
Carbs : Orange in AAPS, Green in Wear watch

=> In xdrip wear companion, colors (high or low BG for example) change when watch face is active or not (colored graph when active, black and white graph when inactive. Of course it's more visible if BG is green on active face...)

**Comment by jamespaulley** [comment]
I have currently worked around this problem by putting my Ticwatch C2 in airplane mode. This puts a small airplane icon on the screen that is filled in when the watch is active and outlined when inactive. Not elegant but at least I can tell visually whether the display is active. I think a subtle visual indicator like that is all that's needed to tell if the watchface is active. This could be a slight reduction in detail or line weight of graphics.

Unification of the colours between the phone app and the watchface would be great too, by the way.

**Comment by jamespaulley** [comment]
> What is most essential data to display on less detailed version?
> 
> * **for sure:** BGV + Trend arrow + measurement age

Yes I agree. BG, trend (graph), measurement age are essential. It wold be good if the user could choose what goes on the ambient display. Perhaps basal graph could be reserved for active display only, but having said that I do value being able to see it at a glance. How much battery life would you realistically save by removing the basal graph?

**Comment by Philoul** [comment]
I made a proposal for Graph Colors in PR https://github.com/MilosKozak/AndroidAPS/pull/2334 if you want to try... (this PR also includes customizable Direct Tap Actions)

![AAPSV2 Wtachface Color proposal](https://user-images.githubusercontent.com/52934600/72840832-645f9200-3c95-11ea-928a-872629bbdeed.png)

Green is lighter than in phone graph because @jotomo told me in https://github.com/MilosKozak/AndroidAPS/pull/2314 that White for in-range readings and current value was used on watch for better readability (in sunlight) over green...

So I made a compromise ...
